.flake8

```diff
@@ -0,0 +1,10 @@
+[flake8]
```

Comment:
We got rid of the `tox.ini` file, so I had to put flake8 settings somewhere else

Reply:
Can we put this in the `pyproject.toml` just to avoid the proliferation of config files?

Reply:
You would be surprised by https://github.com/PyCQA/flake8/issues/234, they really don't want to support `pyproject.toml`! Maybe with https://github.com/john-hen/Flake8-pyproject

Reply:
Done